packages/teleterm/src/ui/TopBar/Connections/ConnectionsFilterableList/ConnectionItem.tsx

```diff
@@ -100,3 +112,18 @@ export function ConnectionItem(props: ConnectionItemProps) {
     </ListItem>
   );
 }
+
+function getKindName(kind: ExtendedTrackedConnection['kind']): string {
+  function assertUnreachable(x: never): never {
+    throw new Error(`Unhandled case: ${x}`);
+  }
```

Comment:
Could you move this out of `getKindName`? :D

Reply:
Done